src/Utilities/tests/test_timer.cpp

```diff
@@ -20,6 +20,15 @@
 
 namespace qmcplusplus
 {
+
+// The manager will take ownership of the timer.  This means the timer must
+// be allocated dynamically, not on the stack.
+void addRawTimer(TimerManagerClass &tm, NewTimer* t)
```

Comment:
This function is added to serve FakeTimer.
FakeTimer is a derived class with extra functions to alter total_time and num_calls for testing.
I feel better remove this class and add testing APIs as friend function to the NewTimer class.
Then we test NewTimer directly instead of FakeTimer and addRawTimer is not needed.

Reply:
Changed